src/views/users/User/UserList.vue

```diff
@@ -65,7 +67,17 @@ export default {
             }
           },
           mfa_level: {
-            width: '130px'
+            width: '130px',
+            formatter: (row) => {
+              const securityMFAAuth = store.getters.publicSettings['SECURITY_MFA_AUTH']
+              if (securityMFAAuth === MFASystemSetting.allUsers) {
+                return this.$t('MFAAllUsers')
+              } else if (securityMFAAuth === MFASystemSetting.onlyAdminUsers && (row?.is_superuser || row?.is_org_admin)) {
+                return this.$t('MFAOnlyAdminUsers')
+              } else {
+                return row['mfa_level'].label
+              }
+            }
           },
           source: {
             width: '120px',
```

Comment:
The code does not contain any errors per my understanding of regular programming conventions in TypeScript and Vue. 

However, there is a potential improvement that can be made to the `source` component's `width` property:

```
export default {
  components: {},
       ... // other properties 
 +    source: {
     width: '50px',   
      /* Other props you want here */
    },  
}.
```